modules/widgets/image-widget.php

```diff
@@ -84,6 +85,7 @@ public function widget( $args, $instance ) {
 			if ( '' != $instance['link'] && empty( $instance['link_target_blank'] ) )
 				$output = '<a href="' . esc_attr( $instance['link'] ) . '">' . $output . '</a>';
 			if ( '' != $instance['caption'] ) {
+				/** This filter is documented in core/src/wp-includes/default-widgets.php */
```

Comment:
Did we settle on this standard for up-linking to core? At least one inline doc is out of standard https://github.com/Automattic/jetpack/blob/202d1b80e9386aa6ed2dceb6ab9a4f362413e695/modules/custom-post-types/testimonial.php#L716